### DISCUSSION TITLE
Provide capability to annotate releases with external artifacts such as NuGet, NPM, Docker Hub, etc 🛳

### DISCUSSION THREAD

**Discussion by tomkerkhove** [community/community#4655] [Actions]
Provide capability to annotate releases with external artifacts such as NuGet, NPM, Docker Hub, etc which are either hosted on GitHub Packages or external package registries.

This would simplify the adoption and provide a better UX.

Relates to #4658 which could use this to pull in adoption metrics.

**Comment by MylesBorins** [comment]
@tomkerkhove this is a great idea and something I've been thinking about quite a bit myself. Will keep this discussion up to date if this is something we decide to tackle

**Reply by tomkerkhove** [reply]
Here's a mock-up:
![Picture1](https://user-images.githubusercontent.com/4345663/136081943-d43d512c-eb03-4a23-b5b6-3bfa9cacb3e1.png)

![Picture3](https://user-images.githubusercontent.com/4345663/136082083-cf20d680-d8a7-4d58-ae89-02b0f0ada81f.png)

**Reply by tomkerkhove** [reply]
Any thoughts on this @MylesBorins ?
